src/chrome/content/rules/Etiqa.com.my.xml

```diff
@@ -0,0 +1,51 @@
+<!--
+	For other Maybank coverage, see Maybank.com.xml
+
+
+	Non-functional subdomains:
+
+		- etiqa.com.my	(h)
+		- www.etiqa.com.my	(t)
+		- bpp	(e)
+		- dev.bpp	(t)
+		- cgs	(r)
+		- cls	(r)
+		- clsgl	(r)
+		- d	(m)
+		- dcatalyst	(m)
+		- edmtrace	(m)
+		- insc1	(t)
+		- insc2	(t)
+		- intranet	(r)
+		- lpappuat1	(t)
+		- partner	(e)
+		- pentaagent	(t)
+	
+	e: expired certificate
+	h: http redirect
+	i: invalid certificate chain
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Etiqa Insurance">
+
+	<target host="academy.etiqa.com.my" />
+	<target host="agentmail.etiqa.com.my" />
+	<target host="media.etiqa.com.my" />
+	<target host="t.etiqa.com.my" />
+	<target host="w.etiqa.com.my" />
+
+	<securecookie host=".+" name=".+" />
+
+	<!-- only MyAccount directory is functional -->
+	<rule from="^http://(www\.)?etiqa\.com\.my/MyAccount"
+		to="https://$1etiqa.com.my/MyAccount" />
```

Comment:
This won't work without having them in the targets.

Reply:
I have to add `exclusion` with `(?!getonline|MyAccount)`.

Reply:
This is already covered by the rule below.